tests/integration/test_context_chunker_removal.py

```
@@ -0,0 +1,336 @@
+    
+    def test_no_imports_of_context_chunker(self):
+        """No Python files should import context_chunker."""
+        python_files = list(REPO_ROOT.rglob("*.py"))
+        
+        for py_file in python_files:
+            # Skip test files and cache
+            if '__pycache__' in str(py_file) or 'test_context_chunker_removal' in str(py_file):
+                continue
+            
+            try:
+                with open(py_file, 'r') as f:
```

Ruff (0.14.6)

Unnecessary mode argument

Remove mode argument

(UP015)